libtextsecure/outgoing_message.js

```diff
@@ -220,7 +242,6 @@ OutgoingMessage.prototype = {
       // TODO: Make NUM_CONCURRENT_CONNECTIONS a global constant
       const options = {
         numConnections: NUM_SEND_CONNECTIONS,
-        isPing: this.isPing,
```

Comment:
Some documentation in PR about what isPing did and why we don't need it any more would be good.